example/script.js

```diff
@@ -0,0 +1,137 @@
+// In this example we are going to be creating two chat clients
+// One will have the unread-messages plugin connected to the global channel
+// The other will not, allowing you to easily see the diff
+
+// get some references to our UI elements
+const rickInput = document.getElementById('rick-input');
+const rickOutput = document.getElementById('rick-output');
+const mortyInput = document.getElementById('morty-input');
+const mortyOutput = document.getElementById('morty-output');
+
+// get some references to functions
+let rickSend = function () {};
+let rickSubmit = function () {};
+let rickActive = function () {};
+let rickInactive = function () {};
+let mortySend = function () {};
+let mortySubmit = function () {};
+
+// here we are creating two instances of chat-engine
```

Comment:
IMO this entire demo should be split into two iFrames so we don't lead newbies in the wrong direction.